DP03_05_Near-Earth_Objects.ipynb

```diff
@@ -0,0 +1,1395 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "749b0ddf",
+   "metadata": {},
+   "source": [
+    "<img align=\"left\" src = https://project.lsst.org/sites/default/files/Rubin-O-Logo_0.png width=250 style=\"padding: 10px\"> \n",
+    "<br><b>CET Template Notebook</b> <br>\n",
+    "Contact author(s): <i>Sarah Greenstreet</i> <br>\n",
```

Comment:
remove italics?

Reply:
Italics removed for consistency. And "CET Template Notebook" changed to "Orbital Properties of Near-Earth Objects (NEOs)".